common/models/acl.js

```diff
@@ -394,7 +396,7 @@ module.exports = function(ACL) {
     var staticACLs = this.getStaticACLs(model.modelName, property);
 
     var self = this;
-    var roleModel = loopback.getModelByType(Role);
+    var roleModel = registry.getModelByType(Role);
```

Comment:
`Role` is the global `loopback.Role`, this should be IMO changed to `getModelByType('Role')`.